app/src/main/java/wordtextcounter/details/main/analytics/AnalyticsConsent.kt

```diff
@@ -28,6 +30,7 @@ object AnalyticsConsent {
     val btSave = dialog.findViewById<Button>(R.id.btSave)
 
     btSave.setOnClickListener {
+      logAnalytics(Click("consent_save"))
```

Comment:
Not sure if GDPR allows even this logging

Reply:
That's safe. If consent wasn't given then Logger won't do anything anyway.

Reply:
Now that I think, this feels redundant. This should be called at the end where we dismiss the dialog. wii update.